src/components/ui/streamer.tsx

```
@@ -0,0 +1,154 @@
+"use client";
+
+import React, { useState, useRef, useEffect } from "react";
+import ReactMarkdown from "react-markdown";
```

ESLint

[error] 'ReactMarkdown' is defined but never used.

(@typescript-eslint/no-unused-vars)